GMPWrapper/python/GMPWrapper/parseConstants.py

```diff
@@ -0,0 +1,22 @@
+import re
+
+def parseConstants(constants):
+  again = True
```

Comment:
Need a docstring for this function. Including a few words of how it does its thing